src/coreclr/tools/aot/crossgen2/crossgen2.props

```diff
@@ -29,28 +29,18 @@
     </EmbeddedResource>
```

Comment:
@jkotas, if we are not planning to add translations in Crossgen2 and ILC, shall we remove these embedded resources? I removed them from ilc. above (it was unused there). I can inline all `SR.` strings in Crossgen2.

Reply:
We have flipped several times on dotnet/runtime tools localization. I would leave it as is. There is non-zero chance that we decide to do that in future.